src/content/docs/status.mdx

```diff
@@ -1,18 +1,15 @@
 ---
-sidebar_position: 8
-title: 👷 Status
+title: Status
 description: Status of the Shorebird project.
 ---
 
-## Stable
-
 Shorebird is available on Android and iOS, with ~1000 apps using Shorebird on
 the App Store and Play Store.
 
 ## Link Percentage (iOS)
 
 As part of [complying with the the App Store
-guidelines](faq#does-shorebird-comply-with-app-store-guidelines), Shorebird uses
+guidelines](/faq#does-shorebird-comply-with-play-store-guidelines), Shorebird uses
```

Comment:
Wrong one?

Reply:
no that's the first of two store links (the only one was removed).